### DISCUSSION TITLE
typescript - enable forceConsistentCasingInFileNames

### DISCUSSION THREAD

**Discussion by BurtHarris** [sapegin/mrm#305] [Ideas]
Microsoft (via webhint) recommends enabling __forceConsistentCasingInFileNames__ in tsconfig.js files.  In https://webhint.io/docs/user-guide/hints/hint-typescript-config/consistent-casing/  they say:

> If you are working on a project where developers use different OSes (e.g., Windows and Linux), enabling this property will prevent issues related to the file casing and how the files are referenced in the code.

I'll prepare a PR as this looks simple to address.   @sapegin I like the idea behind mrm.  Frankly, it looks like you are falling behind on merging PRs, including simple ones.  How can I help?

**Comment by sapegin** [comment]
> Frankly, it looks like you are falling behind on merging PRs, including simple ones. How can I help?

Like it's my job right? ;-)

I'm happy to give you more right in the repository if you have time and desire 👾

**Comment by BurtHarris** [comment]
Sure, that sounds good; make me a contributor (or whatever it takes.)   I am a retired dev and have time to contribute _if_ I can learn from it.   I lack experience in GitHub-based development, and this whole dependabot thing is just bugging the heck out of me.   So, if you can provide some guidance, that satisfies my desire to keep learning.

Q: is there a good way to merge these dependency updates without doing them one at a time?   Pointing me at a web page describing how to do that would be a great answer.   For example, I see many patch-level updates to minimist in various packages in the repo.
